sdks/python/tests/e2e/test_tracing.py

```diff
@@ -408,7 +408,10 @@ def test_manually_created_trace_and_span__happyflow(
         input={"input": "trace-input"},
         output={"output": "trace-output"},
         tags=["trace-tag"],
-        metadata={"trace-metadata-key": "trace-metadata-value"},
+        metadata={
+            "providers": [provider_name],
```

Comment:
Please add a comment that this key is automatically added by the backend from version x.y.z